data/i18n/en/en.toml

```diff
@@ -405,6 +405,9 @@ other = "This article is more than one year old. Older articles may contain outd
 [patch_release]
 other = "Patch Release"
 
+[monthly_patch_release]
+other = "Monthly Patch Release"
+
```

Comment:
Adding this entry _(monthly_patch_release)_ earlier in the file and organizing it alphabetically along with the rest of the entries would be nice.

Reply:
Agreed; alphabetical order of terms please.

Reply:
Fixed as suggested :+1: